lib/collection.js

```diff
@@ -161,12 +161,11 @@ const BookshelfCollection = (module.exports = CollectionBase.extend(
           .bind(this)
           .tap(function(response) {
             if (!response || response.length === 0) {
-              throw new this.constructor.EmptyError('EmptyResponse');
+              if (options.requre === true) {
+                throw new this.constructor.EmptyError('EmptyResponse');
```

Comment:
I previously mixed this and model.js up and had NotFoundError here instead of EmptyError. It was passing tests I found locally now, lets see how circle ci goes.